pkg/vsphere/toolbox/trace_channel.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2016 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package toolbox
+
+import (
+	"encoding/hex"
+	"fmt"
+	"io"
+	"os"
+)
+
+var (
+	Trace = true // TODO: make optional
```

Comment:
I could just move the TraceChannel usage to the example cmd/toolbox/main.go for now.  Was thinking it would be a useful runtime option, at least while developing the toolbox further.